srcpkgs/sdcc/template

```diff
@@ -1,10 +1,12 @@
 # Template file for 'sdcc'
 pkgname=sdcc
 version=4.0.0
-revision=1
+revision=2
 build_style=gnu-configure
+configure_args="--enable-libgc
+ $(vopt_if nonfree --enable-non-free --disable-non-free)"
```

Comment:
`$(vopt_enable nonfree non-free)` should work.

Reply:
Indeed, thanks!